log4j-core/src/main/java/org/apache/logging/log4j/core/LoggerContext.java

```diff
@@ -513,7 +512,7 @@ public Logger getLogger(final String name) {
      * @return a collection of the current loggers.
      */
     public Collection<Logger> getLoggers() {
-        return loggerRegistry.getLoggers().collect(Collectors.toList());
+        return loggerRegistry.getLoggers();
```

Comment:
Shouldn't we rather keep the old defensive copy around to avoid similar problems from integrators?

Reply:
`Collectors.toList()` does make a copy, right?

I could add a test for that in case the implementation of `InternalLoggerRegistry.getLoggers()` changes

Reply:
I'm not concerned about `ILR`, I think it is covered, but integrators providing their own `LR` and such. Anyway... Checked the code and this line used to be `return loggerRegistry.getLoggers()`. :+1: